TrackMyStuffApp/aws-beanstalk-tools-defaults.json

```diff
@@ -0,0 +1,20 @@
+
+{
+    "region" : "us-east-1",
+    "configuration" : "Debug",
+    "framework"     : "net5.0",
+    "self-contained" : false,
+    "application"    : "TrackMyStuff",
+    "environment"    : "TrackMyStuff-dev",
+    "enable-xray"    : false,
+    "enhanced-health-type" : "basic",
+    "additional-options"   : "\"aws:autoscaling:launchconfiguration,SecurityGroups\"=\"sg-6ca61e5e\";\"aws:ec2:vpc,Subnets\"=\"subnet-018f3667\"",
+    "solution-stack"       : "64bit Amazon Linux 2 v2.1.0 running .NET Core",
+    "environment-type"     : "SingleInstance",
+    "cname"                : "trackmystuff-dev",
+    "instance-type"        : "t2.micro",
+    "key-pair"             : "",
+    "instance-profile"     : "aws-elasticbeanstalk-ec2-role",
+    "service-role"         : "aws-elasticbeanstalk-service-role",
+    "proxy-server"         : "nginx"
+}
```

Comment:
This is auto generated by [AWS toolkit](https://aws.amazon.com/visualstudio/). We did a deployment using this tool.